sentences/nb/_common.yaml

```diff
@@ -0,0 +1,29 @@
+language: 'nb'
+responses:
+  errors:
+    no_intent: "Jeg skjønte desverre ikke det"
+    no_area: "Det er ingen områder med navnet {{ area }}"
+    no_domain: "{{ area }} har ingen {{ domain }}"
+    no_device_class: "{{ area }} indeholder ikke {{ device_class }}"
+    no_entity: "Finner ingen enhet med navn {{ entity }}"
+    handle_error: "En uventet feil oppsto"
+lists:
+  color:
+    values:
+      - "blå"
+      - "brun"
+      - "grønn"
+      - "gul"
+      - "hvit"
+      - "lilla"
+      - "oransje"
+      - "rød"
+      - "svart"
+expansion_rules:
+  area: "{area}[et]"
+  name: "{name}[n]"
+  what_is: "hva er"
+  turn: "(skru | slå)"
```

Comment:
The key here is "configuration" right?
I think English makes sense.

Reply:
Besides "area" and "name", I think it would make more sense for the rule names to be in their native language. Some of them may not make sense in different languages, and I wouldn't want people to think they have to translate them.

Reply:
It should be all or none of them.

Reply:
https://github.com/home-assistant/intents/pull/25